.github/workflows/run_linters.yml

```diff
@@ -69,7 +69,11 @@ jobs:
         run: sccache --start-server
       - name: init rust toolchain
         # NOTE: We use nightly Rust only to get nightly fmt & clippy
+        # TODO: use nightly after https://github.com/clap-rs/clap/issues/4733 fixed.
```

Comment:
This should now be fixed https://github.com/rust-lang/rust-clippy/pull/10499

we can try nightly

Reply:
Linters passed.